ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouter.java

```diff
@@ -151,6 +151,9 @@ private OperationController getOperationController() {
     GetBlobOptionsInternal internalOptions = new GetBlobOptionsInternal(options, false, routerMetrics.ageAtGet);
     try {
       BlobId blobId = RouterUtils.getBlobIdFromString(blobIdStr, clusterMap);
+      if (blobId.getDatacenterId() != clusterMap.getLocalDatacenterId()) {
```

Comment:
account for -1 in the `BlobId` (`ClusterMapUtils.UNKNOWN_DATACENTER_ID`) here i.e. -1 should not count towards this rate